xapi-xenopsd-simulator.opam

```diff
@@ -6,15 +6,14 @@ homepage: "https://github.com/xapi-project/xenopsd"
 dev-repo: "https://github.com/xapi-project/xenopsd.git"
 bug-reports: "https://github.com/xapi-project/xenopsd/issues"
 build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-doc: [[ "jbuilder" "doc" "-p" name "-j" jobs ]]
```

Comment:
I think we don't have a `doc` jbuilder command, we have to use 
```
jbuilder build @doc
```
as in the Makefile.